.github/workflows/node-pretest.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Tests: pretest/posttest'
+
+on: [pull_request, push]
+
+jobs:
+  pretest:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ljharb/actions/node/install@main
```

Comment:
I don't have much experience with GitHub Actions, but having code slurped in like this makes me a bit nervous because it is from an external source and it is not pinned to a specific version. Are these legitimate concerns here?

Reply:
Only if you don’t think you can rely on the author maintaining back compat and being a good actor. travis-ci already relies on the same things and auto-updates nvm to whatever version is published.

Happy to peg it to a sha if you prefer, it just means bugfixes or improvements have to be explicitly updated to here rather than automatically flowing in.